manual/manual.adoc

```diff
@@ -239,23 +250,26 @@ monsters with your bare hands. Ammo: None
 image:../sprites/csawa0.png[Ripsaw] |
 1 | Designed for cutting through wood, the ripsaw
 also works well as a melee weapon for cutting through flesh. Ammo: None
-| **Handgun** | 2 | Your starter weapon. A few shots will take down low-level
-monsters but it’s best to upgrade before taking on any tougher opponents.
+| **Handgun** +
+image:../sprites/pista0.png[Handgun] |
+2 | Your starter weapon. Its main purpose is to let you fight your way
+to a better weapon, and to hit shootable switches without wasting a second bullet.
 Ammo: Bullets
 | **Pump-action Shotgun** +
 image:../sprites/shota0.png[Pump-action Shotgun] |
-3 | The main “workhorse” weapon; fires seven pellets
-in a tight cluster and is effective over both short and long ranges against
```

Comment:
I'm okay with this change, but to give some context for the "effective over long range" thing: one of John Romero's design principles when designing Doom's weapons was that no weapon should become entirely useless when picking up a more powerful weapon. The shotgun fires less pellets than the SSG, but has a tighter spread that remains more effective over a long distance.